samples/oauth_php/VetsApiClaims/app/Http/Controllers/ClaimsController.php

```diff
@@ -18,6 +18,51 @@ public function index(Request $request)
             $response = $client->request('GET', $url);
             $claims = json_decode($response->getBody()->getContents(), true);
             return view('claims', ['claims' => $claims['data']]);
-        } 
+        }
+    }
+
+    public function poa_form(Request $request)
+    {
+        return view('poa_form', []);
+    }
+
+    public function poa_form_show(Request $request, $id)
+    {
+        $oauth_session = OauthSession::find($request->session()->get('id'));
+        $url = env('VETS_API_URL') . '/services/claims/v1/forms/2122/' . $id;
+        $client = new Client(['headers' => ['Authorization' => 'Bearer ' . $oauth_session->access_token]]);
+        $response = $client->request('GET', $url);
+        $claim = json_decode($response->getBody()->getContents(), true)['data'];
+        return view('poa_form_show', ['claim' => $claim]);
+    }
+
+    public function poa_form_upload(Request $request, $id)
+    {
+        return redirect()->route('poa_form_show', ['id' => $id]);
+    }
+
+    public function poa(Request $request)
+    {
+        $poaCode = $request->poaCode;
+        $poaFirstName = $request->poaFirstName;
+        $poaLastName  = $request->poaLastName;
+        $oauth_session = OauthSession::find($request->session()->get('id'));
+        $url = env('VETS_API_URL') . '/services/claims/v1/forms/2122';
+        $client = new Client(['headers' => ['Authorization' => 'Bearer ' . $oauth_session->access_token]]);
```

Comment:
I believe these new instances need to be in try/catches and/or setup to 
`'request.options' => array(
     'exceptions' => false,
   )`

but, iirc, this will still throw exceptions on cURL timeouts